packages/cli/src/cli/github/github.ts

```diff
@@ -0,0 +1,168 @@
+import { Env, LogType } from '@basemaps/shared';
+import { Octokit } from '@octokit/core';
+import { Api } from '@octokit/plugin-rest-endpoint-methods/dist-types/types.js';
+import { restEndpointMethods } from '@octokit/plugin-rest-endpoint-methods';
+
+export const owner = 'linz'; // The Owner of the Github repository
+export const repo = 'basemaps-config'; // Github repository name
+export const base = 'master'; // Base head name of repository
+
+export interface Job {
+  imagery: string;
+  tileMatrix: string;
+  content: string;
+}
+
+export interface Blob {
+  path: string;
+  mode: '100644';
+  type: 'blob';
+  sha: string;
+}
+
+export class Github {
+  octokit: Api;
+  logger: LogType;
+
+  constructor(logger: LogType) {
+    this.logger = logger;
+    const token = Env.get(Env.GitHubToken);
+    if (token == null) throw new Error('Please set up github token environment variable.');
+    this.octokit = restEndpointMethods(new Octokit({ auth: token }));
+  }
+
+  isOk = (s: number): boolean => s >= 200 && s <= 299;
+
+  /**
+   * Get branch by name if exists
+   *
+   * @returns {ref} github references or the new created branch
+   */
+  async getBranch(branch: string): Promise<string | undefined> {
+    this.logger.info({ branch }, 'GitHub: Get branch');
+    const response = await this.octokit.rest.repos.getBranch({
```

Comment:
pretty much all of these should fit on one line.

Reply:
Good point. All fixed.